src/expandableCalendar/index.js

```diff
@@ -474,11 +474,22 @@ class ExpandableCalendar extends Component {
   renderKnob() {
     // TODO: turn to TouchableOpacity with onPress that closes it
     return this.props.disablePan ? (
-      <TouchableOpacity style={this.style.knobContainer} onPress={() => this.onKnobPress()}>
-        <View style={this.style.knobContainer} pointerEvents={'none'}>
-          <View style={this.style.knob} testID={CALENDAR_KNOB}/>
-        </View>
-      </TouchableOpacity>
+      <View style={this.style.knobContainer}>
+        <TouchableOpacity
+          activeOpacity={0.75}
+          style={this.style.knobContainer}
+          onPress={() => this.onKnobPress()}>
+          <Image
+            source={
+              isOpen
+                ? this.props.leftArrowImageSource
+                : this.props.rightArrowImageSource
+            }
```

Comment:
isOpen is not defined in this closure, are you sure you gave this a try before you submit ?

Reply:
oh sorry, there's a missing part before the return :

```
const {position} = this.state;
const isOpen = position === POSITIONS.OPEN;
```